packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -3,6 +3,47 @@ title: Okta Identity Engine API Products release notes 2022
 ---
 <ApiLifecycle access="ie" />
 
+## March
+
+### Monthly release 2022.03.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Authentication timestamp is added as an access token claim](#authentication-timestamp-is-added-as-an-access-token-claim) | March 2, 2022 |
+| [Custom Administrator Roles is GA in Production](#custom-administrator-roles-is-ga-in-production) | February 3, 2022 |
+| [Email Address Bounces API is EA in Preview](#email-address-bounces-api-is-ea-in-preview) | March 2, 2022 |
+| [Shareable Authentication Policies](#shareable-authentication-policies) | March 2, 2022 |
+| [Signed request support for generic OIDC IdP is EA in Preview](#signed-request-support-for-generic-oidc-idp-is-ea-in-preview) | March 2, 2022 |
+| [Bug fixed in 2022.03.0](#bug-fixed-in-2022-03-0)         | March 2, 2022           |
+
+#### Authentication timestamp is added as an access token claim
+
+The user-authenticated epoch timestamp is provided as the `auth_time` [claim in the access token](/docs/reference/api/oidc/#reserved-claims-in-the-payload-section).
+
+#### Custom Administrator Roles is GA in Production
+
+The Okta [Custom Administrator Roles](/docs/reference/api/roles/) API provides operations that you can use to create customized roles and assign them to a user or a group.
+
+#### Email Address Bounces API is EA in Preview
+
+Okta admins can now control the bounced email address list through the Email Address Bounces API. When Okta-sent email addresses are blocked from an email service (the bounced email list), admins can use this API to create a list of blocked email addresses to be removed from the email service.
```

Comment:
Should we link to the API docs, to help the devs reading these notes? or should we not if it's only EA in preview?

Reply:
The API docs are missing for this new endpoint. Janet is going to follow up with the Dev Mgr to ensure that we have API docs for this.